manifests/lynis_packages.pp

```diff
@@ -0,0 +1,35 @@
+# === Copyright
+#
+# Copyright 2017, bitvijays
+# Licensed under the Apache License, Version 2.0 (the "License");
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# == Class: os_hardening::lynis_packages
+#
+# Configures Multiple packages as suggested by Lynis: apt-listbugs, apt-listchanges, checkrestart, needrestart, debsecan, debsums, fail2ban,  libapache2-mod-evasive, libapache2-mod-qos, libapache2-mod-security2, arpwatch,
+# arpon, rkhunter, chkrootkit, libpam-tmpdir
+#
+#
+#
+#
+class os_hardening::lynis_packages{
```

Comment:
Here I have the same view like commented in linux-baseline: https://github.com/dev-sec/linux-baseline/pull/59#discussion_r115810181

I personally do not think it makes sense to have it here: fail2ban and some other things usually require configuration and simple installation is not enough. In the same time I think it would be a bit too much if we would start to manage things like fail2ban or arpwatch in our hardening cookbooks/modules/playbooks

any other opinions?